src/components/TeamMemberTasks/TeamMemberTask.jsx

```diff
@@ -124,9 +128,21 @@ const TeamMemberTask = ({
                                 title="Mark as Done"
                                 onClick={() => {
                                   handleMarkAsDoneModal(user.personId, task);
+                                  handleTaskModalOption("Checkmark");	
                                 }}
                               />
                             )}
+                            {hasPermission(userRole, 'removeUserFromTask', roles) && (		
```

Comment:
Not too sure what do you mean by creating a function with a value returned by this function. True and false is returned by this function upon calling it. The function is under the utilities folder and it's in `permissions.js`. I think this is outside of the scope of this PR since changing this function would affect a lot of components that use it as well.

Reply:
sorry, I meant to say a variable, maybe it will help us to be clearer and use it in other parts of the component, something like this:

```suggestion
   const hasRevemoPermission = hasPermission(userRole, 'removeUserFromTask', roles);
   ...
   {hasRevemoPermission && (
   ...
```